### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by fithamlak**
`List structure` requirements implemented.
-  Set up a new project with webpack
-   Create an index.js file and set an array of some simple to-do tasks
-   Write a function to iterate over the tasks array and populate an HTML list item element for 
each task.
-  On page load render the dynamically created list of tasks in the dedicated placeholder
-   The list should appear in order of the index values for each task.
-  Create a style.css and set rules for the To-Do List
-   All your source files (index.html, index.js and style.css) must be located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+### Install
+
+Install this project with 
+
+### Usage
+
+To run the project, execute the following command:
+open your live server
+
```
Comment (elisha2kyakpo1) [inline readme.md:111]: - [ ] The project uses webpack, there are some commands we run to have the project ready on the local machine and show in the browser. Kindly list all the commands users would use to run the project.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @fithamlak, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive commit messages.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+- Write git clone git@github.com:fithamlak/To-do-list.git
+- Go to the repository folder in your command prompt cd To-do-list.
+- Run npm install to install the dev dependencies for linters and webpacks
+
+
+### Install
+
+run only npm install
+
+### Usage
```
Comment (roshan-bajgain) [inline readme.md:112]: - [ ] [ Optional] Please update how a user can start the project. write `npm start` as well.

**Review by roshan-bajgain — APPROVED** [review]
Hi @fithamlak, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions
- [ ] Your commit message should be capitalized. Please be aware next time.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**